cms/djangoapps/contentstore/courseware_index.py

```diff
@@ -627,19 +627,11 @@ def index_about_information(cls, modulestore, course):
 
         # Broad exception handler to protect around and report problems with indexing
         try:
-            if course.catalog_visibility in (CATALOG_VISIBILITY_ABOUT, CATALOG_VISIBILITY_NONE):
-                searcher.remove(cls.DISCOVERY_DOCUMENT_TYPE, [course_id])
-                log.debug(
-                    "Successfully removed %s \"hidden\" course from the course discovery index",
-                    course_id
-                )
-
-            else:
-                searcher.index(cls.DISCOVERY_DOCUMENT_TYPE, [course_info])
-                log.debug(
-                    "Successfully added %s course to the course discovery index",
-                    course_id
-                )
+            searcher.index(cls.DISCOVERY_DOCUMENT_TYPE, [course_info])
+            log.debug(
```

Comment:
shouldn't this be logged as info?

Reply:
yes definitely, thank you for pointing it out